_includes/author-profile.html

```diff
@@ -9,15 +9,34 @@
   {% if author.avatar %}
     <div class="author__avatar">
       {% if author.avatar contains "://" %}
-        <img src="{{ author.avatar }}" alt="{{ author.name }}" itemprop="image">
+        {% assign author_src = author.avatar %}
+        {% assign author_class = "" %}
       {% else %}
-        <img src="{{ author.avatar | absolute_url }}" class="author__avatar" alt="{{ author.name }}" itemprop="image">
+        {% assign author_src = author.avatar | absolute_url %}
+        {% assign author_class = "author__avatar" %}
```

Comment:
I think you can safely remove the `author_class` stuff. Looking back at my original code `class="author__avatar"` on the `<img>` element was left in mistakenly. It's applied on the wrapper `div` and not needed.